components/feature/downloads/src/main/java/mozilla/components/feature/downloads/DownloadsFeature.kt

```diff
@@ -384,7 +384,8 @@ class DownloadsFeature(
         val positiveButtonBackgroundColor: Int? = null,
         @ColorRes
         val positiveButtonTextColor: Int? = null,
-        val positiveButtonRadius: Float? = null
+        val positiveButtonRadius: Float? = null,
+        val fileNameEndMargin: Int? = null
```

Comment:
I'm completely sure if the `fileNameEndMargin` is something that apps should be customizing. Sounds more like something that AC should determine.

Reply:
Could you help me to clarify? :)

Reply:
Okay, here it comes! `shouldWidthMatchParent` parameter allows users of the feature to display the dialog fullscreen. Fenix project does that. There is no issue when the dialog is not in fullscreen mode, but there is when it's fullscreen.

So, the issue with that is that we sometimes end up having a textview sticking right to the edge of the screen. But usually there is always some marging there. Some phones have curved screens, screenprotectors might hide the very edge, it's part of the design, but there is always some margin between the edge of the screen and the text.

Adding end margin would also make it more consistent with the [next dialog](https://github.com/mozilla-mobile/fenix/pull/22325) fenix displays, when the downloading is finished. It has some end margin. Consistency is usually nice,

Reply:
If margin is needed shouldn't we add it by default?

Reply:
It is not always the case that `shouldWidthMatchParent = true` means full screen though, right? Also, wouldn't it be adding unwanted side effects to a parameter? `shouldWidthMatchParent` doesn't mean `fullscreen`, so only fenix project knows that it is using the view in full screen mode. It kinda makes sense to allow it to decide if it would like to adjust it parameters, in the same way `positiveButtonRadius` work.

Reply:
Got it thanks for the clarification 👍🏽